src/components/data/TableContext.ts

```diff
@@ -1,13 +1,20 @@
 import { createContext } from 'preact';
 import type { RefObject } from 'preact';
 
-export type TableInfo = {
+export type CommonTableInfo = {
   /** This table has click-able, focus-able rows */
-  interactive: boolean;
+  interactive?: boolean;
   /** This table has a sticky header */
-  stickyHeader: boolean;
+  stickyHeader?: boolean;
   /** Turn off outer table borders */
-  borderless: boolean;
+  borderless?: boolean;
+  /** Show a different background every other row */
+  striped?: boolean;
+  /** Show grid lines around table cells */
+  grid?: boolean;
+};
```

Comment:
As noted elsewhere, this is an internal type that isn't exported, so I wouldn't use it to define `TableProps`. Also it isn't immediately obvious what belongs in `CommonTableInfo` vs `TableInfo`.